Samples/UIWidgetsSamples_2019_4/Assets/Script/ModelViewHelper.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using Unity.UIWidgets.foundation;
+using UnityEngine;
+
+namespace UIWidgetsSample
+{
+    public class ModelViewState
+    {
+        public RenderTexture _renderTexture;
+        public GameObject _modelViewPrefab;
+    }
+    
+    public class ModelViewHelper : MonoBehaviour
+    {
+        static ModelViewHelper _instance;
+
+        public static ModelViewHelper Instance
+        {
+            get
+            {
+                if (_instance == null)
+                {
+                    //ensureModelViewHelperIfNeeded();
+                }
+
+                return _instance;
+            }
+        }
+        
+        /*internal static void ensureModelViewHelperIfNeeded() {
```

Comment:
According to this design, user who want to use this function must add `ModelViewHelper.cs` to one GameObject by himself at first. I'm not sure if this is a nice way. Maybe we could do this operation in a function such as `InitModelViewHelper()` and call it before using `Texture` Widget

Reply:
In this PR, the `ModelViewHelper` is more likely a temporary utility we added here to support the demo, i.e., `3DTest1`, instead of a universal module for UIWidgets. That's why I put it inside the Sample project instead of the UIWidgets package.

However, let me refine this class a bit and add some comments to make this more clear